### PR TITLE
Introduce Storage Endpoints

### PR DESCRIPTION
**Pull request by jankodera-ps**
Tenant Network concept is being remodeled in the underlying Python SDK and functionality is being moved to Storage Endpoints + Network Interface Groups. The PR mirrors these changes.

##### SUMMARY
Introduces Storage Endpoints (module fusion_se) and deprecates Tenant Networks (module fusion_tn).

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
fusion_se, fusion_tn

##### ADDITIONAL INFORMATION
This mirrors changes in underlying Python SDK that lives in PyPI 'purefusion' package.

### CODE REVIEW

**Comment by sdodsley** [comment]
Please lint against `black`

File: plugins/modules/fusion_se.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2022, Simon Dodsley (simon@purestorage.com)
```
Comment (sdodsley) [inline plugins/modules/fusion_se.py:4]: nit: Please change the date and use your name

**Review by sdodsley — CHANGES_REQUESTED** [review]
Don't change `galaxy.yml` until we do a PR for the creation of the next release.
Otherwise this looks good.

File: plugins/modules/fusion_se.py (hunk 1 above)
Comment (jankodera-ps) [reply plugins/modules/fusion_se.py]: Fixed

**Review by sdodsley — APPROVED** [review]

**Pull request merged**